### ISSUE TITLE
Url Record seName and Picture seoFileName different validation

### ISSUE THREAD

**Issue by kadri1914** [nopSolutions/nopCommerce#7338]
nopCommerce version: 4.60.2

Steps to reproduce the problem: 
The seName validation and max lenght edits made in the ValideSeNameAsync function in UrlRecordService are not made for product images.
In ValideSeNameAsync function, seName is edited to get maximum 200 characters (NopSeoDefaults.SearchEngineNameLength).

In picture add/update functions in PictureService, seoFileName is edited to take 100 characters. Also nvarchar(300) in the database.

**Comment by AndreiMaz** [comment]
@kadri1914 It's by design. File length is limited to this length (it's not URL). Does it cause any issue? If you experience any issue with picture names, please describe in more details

**Comment by kadri1914** [comment]
Hi @AndreiMaz ,
The problem I faced is a little different. I created a scheduled task that reads the products in the XML address from outside and imports them. Here I used seoFileName to match the product picture names in the xml. When the scheduled task ran 5-6 times, I realized that it added some pictures more than once. While searching, I realized that the value I send as picture file name is limited to 100 characters. The image name from XML is 119 characters.
